scripts/migrate-ideas.mjs

```
@@ -0,0 +1,77 @@
+import { parse } from 'csv-parse/sync'
+import { readFileSync, writeFileSync } from 'fs'
+import { slug } from 'github-slugger'
+import { join } from 'path'
+
+// Read the CSV file
+const csvContent = readFileSync('One Dollar Apps Ideas.csv', 'utf-8')
+
+// Parse CSV content
+const records = parse(csvContent, {
+    columns: true,
```

ESLint

[error] Delete `··`

(prettier/prettier)